### PR TITLE
Home and imported Html 

### PR DESCRIPTION
**Pull request by saketkum11**
Lists 
- Imported html pages.
- Added assets.
- Added component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,24 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import {Routes,Route} from "react-router-dom";
+import Home from "./Pages/Home/Home";
+import Product from "./Pages/Product/Product";
+import Wishlist from "./Pages/Wishlist/Wishlist";
+import Cart from "./Pages/Cart/Cart";
+import Login from "./Pages/Auth/Login";
+import Signup from "./Pages/Auth/Signup";
```
Comment (znjs) [inline src/App.js:9]: Suggestion: by using barrel import all these components can be Imported in one line from `./Pages`

File: src/Component/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    <nav class=" bg-green-7 text-color-0 pd-5 position-fixed t-0">
+        <div class="wt-80  m-auto  flex items-center">
+            <div class="text-xm">
+                <Link class="text-dec text-color-0 text-m cursor" to="/">Farmgistic</Link>
+            </div>
+            <div class="flex items-center justify-even m-lf">
+                <input type="search" placeholder="search"
+                    class="pd-y-3 pd-x-5 rounded-L border-none text-center outline-none" />
+                <i class="fa-solid fa-magnifying-glass bg-black-0 text-color-9 rounded-L pd-3 text-xm"></i>
+            </div>
```
Comment (znjs) [inline src/Component/Navbar/Navbar.jsx]: Replace attribute `class` with `className` in all occurrences in  this file

File: src/Pages/Auth/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  
+                  </div>
+                  <div class="wt-100">
+                      <button
+                          class="cursor bg-red-7 text-color-0 text-xm outline-none border-none pd-x-11 pd-y-3 rounded-xs wt-100">Sign
+                          Up</button>
+                  </div>
+                  <div class="m-y-3">
+                      <a href="/component/login/login.html" target="_blank" class="text-dec text-color-9">Already have
+                          account?</a>
```
Comment (znjs) [inline src/Pages/Auth/Signup.jsx]: Suggestion: use `<Link>` instead of `a` tag

**Review by znjs — APPROVED** [review]
Good Folder structure

change all occurances of `class` to `className` and `for` to `htmlFor`

you can add eslint so these warning can be highlighted

File: src/Pages/Auth/Signup.jsx (hunk 3 above)
Comment (saketkum11) [reply src/Pages/Auth/Signup.jsx]: Thank you .
this was page to import the html only .so missed out this.

File: src/Component/Navbar/Navbar.jsx (hunk 2 above)
Comment (saketkum11) [reply src/Component/Navbar/Navbar.jsx]: Thanks for reminding me , actually forgot to replace. sure will add.

File: src/App.js (hunk 1 above)
Comment (saketkum11) [reply src/App.js:9]: This is new way. ok will sure do this.

**Review by shubhambishnoi — APPROVED** [review]

**Pull request merged**